plugin-ci/orb.yml

```diff
@@ -104,11 +104,22 @@ jobs:
       - checkout
       - *restore_cache
       - run: make dist
+      - run:
+          name: "Generating release notes"
+          command: |
+            echo "### Supported Mattermost Server Versions: $(cat plugin.json | jq .min_server_version -r)+"  >> release-notes.md
+            if [[ $(git tag -l | wc -l) -eq 1 ]]; then
+              git log --pretty=oneline --abbrev-commit --no-decorate --no-color $(git rev-list --max-parents=0 HEAD) HEAD >> release-notes.md;
+            else
+              git log --pretty=oneline --abbrev-commit --no-decorate --no-color tags/$(git describe --tags --abbrev=0)^..HEAD >> release-notes.md;
```

Comment:
I'm not sure if this is correct. At the point this is run, there is already a tag pointing to (most likely) the same commit as `HEAD`. `git describe --tags --abbrev=0` will just return this commit.

Reply:
There is a mistake while getting the previous tag. The correct command is `git log --pretty=oneline --abbrev-commit --no-decorate --no-color $(git describe --tags --abbrev=0 $(git describe --tags --abbrev=0)^)..HEAD >> dist/release-notes.md;`

The caret symbol fetches the second last tag and compares with HEAD which will have the latest tag.